src/s_audio_jack.c

```diff
@@ -186,11 +187,18 @@ static int jack_xrun(void* arg) {
   return 0;
 }
 
+int max_client_name_size(void) {
```

Comment:
this should be `static` (in order to not expose it via the ABI).
but read the comment about it's use below...